services/postmark/config.json

```diff
@@ -0,0 +1,67 @@
+{
+  "config": {
+    "name": "postmark",
+    "label": "Postmark",
+    "description": "Postmark is a transactional email service to enable your app to reliably send and receive email.",
+    "category": "email",
+    "subdomain-required": false
+  },
+  "fields": [
+    {
+      "name": "dkim_hostname",
+      "label": "DKIM Hostname",
+      "description": "The value from the DKIM 'Hostname' column in your DNS Settings for this domain.",
+      "append": "",
```

Comment:
Please just omit this key when you do not need to append a domain name for a field.